strawberryfields/backends/fockbackend/ops.py

```diff
@@ -27,7 +27,11 @@
 from scipy.special import factorial as fac
 from scipy.linalg import expm as matrixExp
 
-from thewalrus.fock_gradients import Dgate, Sgate, S2gate, BSgate
+# from thewalrus.fock_gradients import Dgate, Sgate, S2gate, BSgate
```

Comment:
Just for my own understanding, these imports depend on the corresponding PR in The Walrus, right? https://github.com/XanaduAI/thewalrus/pull/164

Reply:
Correct. Also, I am not sure the names I used are going to be the definitive ones.